docs/index.html

```diff
@@ -4,188 +4,188 @@
 ds: 6
 ds_map:
     alert-dialog:
-        name: alert-dialog
-        version: 2.1
+        ds-name: alert-dialog
+        ds-version: 2.1
     avatar:
-        name: avatar
-        version: 1.0
+        ds-name: avatar
+        ds-version: 1.0
     badge:
-        name: badge
-        version: 1.2
+        ds-name: badge
+        ds-version: 1.2
     breadcrumbs:
-        name: breadcrumb
-        version: 1.2
+        ds-name: breadcrumb
+        ds-version: 1.2
     button:
-        group: button
-        name: button
-        version: 1.6
+        ds-group: button
+        ds-name: button
+        ds-version: 1.6
     carousel:
-        name: carousel
-        version: 1.1
+        ds-name: carousel
+        ds-version: 1.1
     calendar:
-        name: calendar
-        version: 1.0
+        ds-name: calendar
+        ds-version: 1.0
     checkbox:
-        name: checkbox
-        version: 2.0
+        ds-name: checkbox
+        ds-version: 2.0
     chip:
-        version: 1.0.0
+        ds-version: 1.0.0
     chips-combobox:
-        name: chip-combobox
-        version: 1.0.0
+        ds-name: chip-combobox
```

Comment:
Last I checked, DS was using `input-chips`. If this is still the case, let's change it. Can you please check?

Reply:
@randybascue, can you confirm we want to use `chip-combobox` as the DS name for this component?

Reply:
We actually use [`chips-combobox`](https://opensource.ebay.com/ebayui-core/?path=/docs/form-input-ebay-chips-combobox--documentation) in eBayUI, not sure if that effects the decision.

Reply:
We're updating most to be singular instead of plural, so input-chip and chip-combobox

Reply:
@ianmcburnie , since most of this appears to be for DS and we're going to have many differences with Skin, should we broaden this schema and include the skin submodules used in each module? Skin's structure added, this would look a bit like this:

```
chips-combobox:
    name: chip
    ds_name: input-chip
    version: 1.0.0
    ...
    name: combobox
    ds_name: combobox
    version: 1.0.0
    ...
```

If there's little value in doing this, I'm ok skipping it, but I thought I'd at least mention it.

Reply:
Yeah, I do kind of like the idea. Maybe just one extra field containing an array? @cordeliadillon what do you think?

```
chips-combobox:
   ds-name: input-chip
   version: 1.0.0
   submodules: [chip, combobox]
```

Reply:
Adding submodules makes sense!

@ianmcburnie, if we just had an array of submodules, would that imply that they collectively follow the version of the parent module? Or would it be more like a reference to the each of those modules, which in turn might have their own versions defined?

Reply:
The latter I would say. Although it gets interesting because on the engineering side we separated out chip as a top level module, but design didn't. So we are just giving it the same version number as input-chips. The other part of input-chips, the combobox, _does_ have it's own independent version in design system.

Reply:
Yea, I'd say the latter too. Since we typically create a module from portions of module specs that can be used individually, I'd say every submodule can be a parent module, but their versioning can be defined on their own parent module section.

Reply:
Working on this now! Here's how it's looking:

```yaml
 chip:
      ds-name: input-chip
      ds-version: 1.0.0
  chips-combobox:
      ds-name: chip-combobox
      ds-version: 1.0.0
      submodules:
      - chip
      - combobox
  combobox:
      ds-name: dropdown
      ds-version: 2.2
      submodules:
      - icon
      - listbox
      - textbox
  ```

I do have a question, prompted by `lightbox-dialog`. This Skin module maps to two Design System patterns, so I was using an array to show its design system components:

```yaml
lightbox-dialog:
    - ds-name: modal
      ds-version: 2.1
    - ds-name: bottom-sheet
      ds-version: 2.1
```

Adding `submodules` changes `lightbox-dialog` from an array of objects to an object with 2 arrays:
```yaml
lightbox-dialog:
    ds-components:
    - ds-name: modal
      ds-version: 2.1
    - ds-name: bottom-sheet
      ds-version: 2.1
    submodules:
    - button
    - icon-button
 ```

... which makes me wonder, should we modify the schema so that _each_ module takes this format? Could allow this schema to evolve more easily over time.
```yaml
skin-module-name:
  ds-components:
  - name: // Design System component name
    version: // Design System version implemented
  - ...
  submodules:
  - submodule-name-1 // Skin module name
  - submodule-name-2 // Skin module name
  - ...
```

e.g.

```yaml
chip:
    ds-components:
    - name: input-chip
      version: 1.0.0
chips-combobox:
    ds-components:
    - name: chip-combobox
      version: 1.0.0
    submodules:
    - chip
    - combobox
combobox:
    ds-components:
    - name: dropdown
      version: 2.2
    submodules:
    - icon
    - listbox
    - textbox
  ```

Reply:
@ianmcburnie @ArtBlue In my latest commit, I added two schemas to consider. I think the second one is more flexible, but defer to you all on what's best for Skin.

i also did a pass over the submodules by manually reviewing each module's HTML markup examples. I've double checked but would appreciate your review in case there's anything I missed. The submodules are the same in both schema options.

Reply:
@cordeliadillon , I like the second one better too. The top-level split between ds and skin makes it easier to scan as well.